### ISSUE TITLE
Headers are not installed in the include directory

### ISSUE THREAD

**Issue by tytan652** [aja-video/libajantv2#1]
With a prefix set to `/usr`, no headers are installed in `/usr/include` to allow building application against libajantv2.

**Comment by paulh-aja** [comment]
We've made an issue for this in our internal issue tracker and will have a fix merged shortly.

**Comment by thomas43** [comment]
@tytan652 this should be resolved in commit 8ee65b987da0185693377c28c2e6721cdf6b563a in `ntv2_17_0` and commit 20a5c85560e68be9fe07752a608ec413931616d8 in `main`. Please close the issue if it's resolved on your end.

**Comment by tytan652** [comment]
Unfortunately, there is still headers that are installed in `/usr/libajantv2` with subfolders:
- `/usr/libajantv2/ajantv2`
  - `/usr/libajantv2/ajantv2/includes`
  - `/usr/libajantv2/ajantv2/src/lin/ntv2linuxpublicinterface.h`
- `/usr/libajantv2/driver`
  - `/usr/libajantv2/driver/linux`
  
My CMake command for packaging testing:
``` sh
cmake -B build -S libajantv2 \
    -DCMAKE_BUILD_TYPE=None \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DCMAKE_INSTALL_LIBDIR=lib \
    -DAJANTV2_BUILD_OPENSOURCE=ON \
    -DAJANTV2_BUILD_SHARED=ON \
    -DAJANTV2_DISABLE_DEMOS=ON \
    -DAJANTV2_DISABLE_DRIVER=ON \
    -DAJANTV2_DISABLE_TESTS=ON \
    -DAJANTV2_DISABLE_TOOLS=ON \
    -DAJANTV2_DISABLE_PLUGINS=ON \
    -DAJA_INSTALL_HEADERS=ON \
    -DAJA_INSTALL_SOURCES=OFF \
    -DAJA_INSTALL_CMAKE=OFF \
    -DAJA_INSTALL_MISC=OFF \
    -DAJA_DISABLE_QT=ON \
    -DAJA_DISABLE_AMD=ON \
    -DAJA_DISABLE_NVIDIA=ON \
    -Wno-dev
```

**Comment by thomas43** [comment]
I agree, those extra headers should not be installed, you have `AJANTV2_DISABLE_DRIVER=ON`

Should be fixed in commit e2f93530 on `main` and commit d5f59e3f in `ntv2_17_0`

**Comment by tytan652** [comment]
Fixed but I'm waiting for a new release to consider it fixed.
